src/gss_sec_ctx.c

```diff
@@ -1306,6 +1332,8 @@ uint32_t gssntlm_inquire_sec_context_by_oid(uint32_t *minor_status,
         return gssntlm_spnego_req_mic(minor_status, ctx, data_set);
     } else if (gss_oid_equal(desired_object, &sasl_ssf_oid)){
         return gssntlm_sasl_ssf(minor_status, ctx, data_set);
+    } else if (gss_oid_equal(desired_object, &get_sids_oid)){
```

Comment:
Space before open brace.  (Yes, I'm aware that the lines above don't adhere to this.)